test/functional/lua/vim_spec.lua

```diff
@@ -1453,3 +1453,18 @@ describe('lua stdlib', function()
     end)
   end)
 end)
+
+describe('lua: require("mod") from packages', function()
+  before_each(function()
+    command('set rtp+=test/functional/fixtures')
+  end)
```

Comment:
👎

Reply:
there is a clear in the `before_each` for the entire file.

Reply:
sorry, I need to clarify

the previous description and the nested context for the specific error case was clearer than the current version:
 - the nested context described how we build it(why we need `before_each`)
 - it could be easier extended with other scenarios(like a "happy-path" cases)
 - it was consistent with other test cases in this spec

moreover, the current description is kinda "verbose":
 - "lua": looks like unnecessary prefix here
-  "require from packages"? could it be different `require`?